pages/04.Reference/04.block-storage/docs.en.md

```diff
@@ -10,35 +10,112 @@ taxonomy:
+
+The transfer can be initiated in one project using `openstack volume transfer request create`, and then completed using `openstack volume transfer request accept`.
+
+### Volume snapshots
+
+A snapshot saves a point-in-time copy of a block storage volume.
+
+```
+openstack volume snapshot create --volume volume_name_or_id snapshot_name
 ```
 
----
+Snapshots can only be created while the volume is detached. The option `--force` to create a snapshot from an attached volume is currently not supported and may lead to a snapshot in ERROR state.
 
-## Questions & Answers
+You may use the snapshot as a source when creating a new volume:
 
-### Can I use a block storage volume across multiple VMs simultaneously?
+```
+openstack volume create --snapshot snapshot_id name_of_new_volume
+```
+
+### Save volume as glance image
+
+It is possible to copy the contents of a cinder volume to the [glance image store](../06.images/docs.en.md), e.g. using the [OpenStack CLI](../../02.Tutorials/02.api-access/docs.en.md):
+
+```
+openstack image create --volume volume_name_or_id name_of_new_image
+```
+
+The cinder volume must be detached for the operation to succeed.
+
+### Multi-attach volumes
+
+When you create a volume with a multi-attach volume type (See <a href="#Available volume types">available volume types</a>), it is possible to attach it to multiple virtual machines at the same time.
 
-Unfortunately this is not possible. Think of a block storage volume like a virtual hard disk - just as you cannot connect a hard disk to multiple computers at the same time, you cannot connect a block storage volume to multiple compute instances at the same time.
+Please refer to our multi-attach volume tutorial. It explains how to use this feature with the cluster file system ocfs2. TODO: Add link to tutorial.
 
-### How can I get shared storage inside SysEleven Stack?
+!! WARNING: This mode of operation requires special cluster file systems like ocfs2. Otherwise it can lead to the loss of data and / or file system and data corruption.
 
-There are multiple options, for instance:
+### Volume resizing
+
+Sometimes it is necessary to change the size of your volume.
+
+Cinder volumes can be extended in size using the API or the OpenStack CLI:
+
+```
+openstack volume set --size XX volume_name_or_id
+```
+
+Following the volume size change it might be necessary to grow the filesystem. If you are using ext4 you may need to run this command on the VM where the volume is attached:
+
+```
+resize2fs /dev/vdX
+```
 
-* S3 (SEOS) - provided as part of the SysEleven Stack
-* NFS, GlusterFS, Ceph - interim solution for legacy applications
+### Current limitations
 
-For the NFS case, we provide a [shared volume](https://github.com/syseleven/heat-examples/tree/master/shared-volume) template with multiple automatically connecting NFS clients, which you can use with our orchestration service.
```

Comment:
I miss this section. We should still mention NFS, GlusterFS, Ceph & Co. as alternative approaches to multiattach and refer to our shared volume nfs example.

Reply:
For me, bodo and dennis it does not feel right to link the nfs example in the block storage docs. Should we write a background article that compares all available storage options with lots of background information, we could link the background article here and on other reference guides (e.g. compute for local storage vs. cinder storage etc)

Reply:
ok